Examples/Swift/ManageOfflineRegionsExample.swift

```diff
@@ -0,0 +1,220 @@
+import Mapbox
+
+@objc(ManageOfflineRegionsExample_Swift)
+
+class ManageOfflineRegionsExample_Swift: UIViewController, MGLMapViewDelegate, UITableViewDelegate, UITableViewDataSource  {
+
+        var mapView: MGLMapView!
+        var progressView: UIProgressView!
+        var downloadButton = UIButton(frame: CGRect.zero)
+        var tableView = UITableView()
+        var regionCount = 0
+        var regionArray = [MGLTilePyramidOfflineRegion]()
+        var packArray = [MGLOfflinePack]()
+        var userInfo = [String]()
+        var pack = MGLOfflineStorage.shared.packs
+        var progressPercentage: Float?
```

Comment:
Instead of declaring this here, perhaps consider extending `MGLOfflinePackProgress` to have a computed property that returns the progress percentage. 

By doing this, the responsibility of calculating the progress of the pack moves from the `ViewController` class to the `MGLOfflinePackProgress` object itself. 

You don't need to add the extension to another file, just include it at the bottom of the file and make sure its marked as `fileprivate`, so it can't be accidentally used in future examples.